cloudbridge/cloud/providers/gce/services.py

```diff
@@ -410,16 +427,20 @@ class GCEInstanceService(BaseInstanceService):
     def __init__(self, provider):
         super(GCEInstanceService, self).__init__(provider)
 
-    def create(self, name, image, instance_type, zone=None,
+    def create(self, name, image, instance_type, network=None, zone=None,
                key_pair=None, security_groups=None, user_data=None,
-               launch_config=None,
-               **kwargs):
+               launch_config=None, **kwargs):
         """
         Creates a new virtual machine instance.
         """
         if not zone:
             zone = self.provider.default_zone
         if not launch_config:
+            if network:
+                network_url = (network.resource_url
```

Comment:
I think these short forms are good when you can fit the whole thing in one line; otherwise, a complete if statement is cleaner, IMHO. Feel free to ignore, specially since it looks like there are statements like this in other parts of the code.